pyresample/bucket/__init__.py

```diff
@@ -266,14 +277,14 @@ def get_min(self, data, fill_value=np.nan, skipna=True):
             Bin-wise minimums in the target grid
         """
         LOG.info("Get min of values in each location")
-        return self._call_pandas_groupby_statistics('min', data, fill_value, skipna)
+        return self._call_bin_statistic('min', data, fill_value, skipna)
 
     def get_max(self, data, fill_value=np.nan, skipna=True):
         """Calculate maximums for each bin with drop-in-a-bucket resampling.
 
-        .. warning::
+        .. note::
 
-            The slow :meth:`pandas.DataFrame.groupby` method is temporarily used here,
+            The :meth:`numpy.ufunc.reduceat`  method is temporarily used here,
```

Comment:
It seems that `numpy.ufunc.reduceat` isn't used anymore, so the note can be removed.

Reply:
Fixed.